exercises/03_count_byte/countb.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func CountByte(s string, b byte) int {
+	var i int
+	j := []byte(s)
+	for a := range j {
+		if j[a] == b {
+			i++
+		}
+	}
+	return i
+}
+
+func main() {
+	r := bufio.NewReader(os.Stdin)
+	scanner := bufio.NewScanner(r)
+	fmt.Print("Enter a string: ")
+	scanner.Scan()
+	s := scanner.Text()
+	fmt.Print("Enter a byte: ")
+	var b byte
+	fmt.Fscanf(r, "%c", &b)
+	fmt.Printf("s = %[1]q\nb = %[2]c (%[2]d)", s, b)
+	fmt.Print(" => contains ")
+	fmt.Printf("%[2]c", s, b, " ")
```

Comment:
This and next line look weird. It's just a wrong formatting or were you just experimenting here? I would do smth like this:
```go
if cnt := CountByte(s, b); cnt > 0 {
  fmt.Printf("%[1]q contains byte %[2]c (code %[2]d) %[3]d times.\n", s, b, cnt)
} else {
  fmt.Printf("%[1]q does not contain byte %[2]c (code %[2]d).\n", s, b)
}
```
This way the output string is very clear